### PR TITLE
Update the cassandra example to pick up recent changes.

### PR DESCRIPTION
**Pull request by brendandburns**
Also make it buildable via Maven.

@mikeln @roberthbailey

### CODE REVIEW

**Comment by mikeln** [comment]
I won't be able to get to this until tonight.

File: examples/cassandra/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ desiredState:
     id: cassandra
     containers:
       - name: cassandra
-        image: kubernetes/cassandra
+        image: kubernetes/cassandra:v2
```
Comment (roberthbailey) [inline examples/cassandra/README.md]: Thank you for adding explicit tags.

**Comment by roberthbailey** [comment]
LGTM, so going to merge on green. @mikeln If you have any comments, please send @brendandburns and/or myself a PR to clean this up.

**Comment by mikeln** [comment]
Tested for fix of Cassandra Cluster Example Custom Seed Cores on First Node #5737
no error anymore.

LGTM

**Comment by roberthbailey** [comment]
Fixes #5737.

**Pull request merged**